importexport/musicxml/exportxml.cpp

```diff
@@ -2403,21 +2419,38 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
       // and finally the attributes whose elements are children of <technical>
       for (const Articulation* a : na) {
             auto sid = a->symId();
+            QString placement = "";
```

Comment:
@eduardomourar could you please make it simpler using just a `QString placement;` statement? Here and below.